docs/DeveloperGuide.md

```diff
@@ -521,69 +524,65 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 **MSS**
 
 1.  User enters the details of volunteer to be added into the application.
-2.  FL adds the volunteer into the database, and feedbacks the successful addition of the volunteer.
-3.  User see the volunteer details appear in the volunteer list.
+2.  FL feedbacks the successful addition of the volunteer and shows the new volunteer.
```

Comment:
all below same

Reply:
Resolved in [a108200](https://github.com/AY2223S2-CS2103T-W12-1/tp/pull/294/commits/a1082000ca6232ebe7eed75ff3d81f9e2a2dac2a)